chapters/background.tex

```diff
@@ -1,4 +1,11 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+TensorFlow is a open source Machine Learning library developed by Google.
```

Comment:
I also thought that things such as "Machine Learning" or "Computer Vision" have to be capitalized, but after some research, it seems that this is not the case. So, I'd suggest to write "machine learning".

EDIT: Some sources are saying that it does not have to capitalized. Honestly, we should just make sure it's consistent within our report, regardless of whether it's capitalized or not.